docs/README.md

```diff
@@ -0,0 +1,320 @@
+# rules_js documentation
+
+> Migrating from rules_nodejs? Start at the [Migration Guide](./migrate.md)
+
+Stuck?
+
+-   See the [Frequently asked questions](./faq.md)
+-   Ask in `#javascript` on <http://slack.bazel.build>
+-   Check for [known issues](https://github.com/aspect-build/rules_js/issues)
+-   Pay for support, provided by <https://aspect.dev>.
+
+## Installation
+
+From the release you wish to use:
+<https://github.com/aspect-build/rules_js/releases>
+copy the WORKSPACE snippet into your `WORKSPACE` file.
+
+Note that [bzlmod] is experimentally supported as well.
+
+## Usage
+
+### Bazel basics
+
+Bazel's `BUILD` or `BUILD.bazel` files are used to declare the dependency graph of your code.
+They describe the source files and their dependencies, and declare entry points for programs or tests.
+However, they don't say _how to build_ the code, that's the job of Bazel rules.
+
+Because `BUILD` files typically declare a finer-grained dependency graph than `package.json` files, Bazel can be smarter about what to fetch or invalidate for a given build.
+For example, Bazel might only need to fetch a single npm package for a simple build,
+where you might experience other tools installing the entire `package.json` file.
+
+On the other hand, authoring BUILD files is a chore that's required under Bazel and not in
+other tools like Rush or Nx. We plan a [Gazelle] extension soon which automates much of this toil.
+
+Other recommendations:
+
+-   Put [common flags](https://blog.aspect.dev/bazelrc-flags) in your `.bazelrc` file.
+-   Use [Renovate](https://docs.renovatebot.com/) to keep your Bazel dependencies up-to-date.
+
+### Node.js
+
+rules_js depends on rules_nodejs version 5.0 or greater.
+
+Installation is included in the `WORKSPACE` snippet you pasted from the Installation instructions above.
+
+**API docs:**
+
+-   Installing and choosing the version of Node.js:
```

Comment:
In my understanding, the `nodejs` toolchain is installed in the `WORKSPACE` snippet above, we don't need to `install` nodejs ourselves. Is that correct?

Reply:
that's right. I'll just label this one "choosing the version..."